src/main/resources/org/jenkinsci/plugins/ghprb/GhprbTrigger/help-blackListCommitAuthor.html

```diff
@@ -0,0 +1,3 @@
+<div>
+	When filled, pull request commits from this user(s) will <b>not</b> trigger a build.
```

Comment:
Could you add a bit more, pointing out that this will be overridden if a job's config is using blacklist commit authors?

Maybe a second sentence like:
`(Note: this can be overridden by job-specific configuration.)`